NDTensors/src/lib/GPUArraysCoreExtensions/test/runtests.jl

```diff
@@ -0,0 +1,7 @@
+@eval module $(gensym())
+using Test: @testset, @test
+using NDTensors.GPUArraysCoreExtensions
```

Comment:
```suggestion
using NDTensors.GPUArraysCoreExtensions: storagemode
```
and same thing elsewhere. We should use that style of explicitly listing functions being brought into the namespace in general.

Reply:
Okay i have updated this across the library, thanks